nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -383,6 +391,19 @@ in
       '';
       example = "/mnt/nextcloud-file";
     };
+    extraSecrets = lib.mkOption {
```

Comment:
The other module I'm aware of that does this is forgejo and it calls this option `secrets`. Should we do the same?
Kinda unfortunate that this isn't covered in rfc42.

Reply:
:+1: from me, the `extra` part doesn't help much IMO.